app/view_models/prior_authority/v1/application_details.rb

```diff
@@ -3,32 +3,59 @@ module V1
     class ApplicationDetails < ApplicationSummary
       attribute :ufn, :string
       attribute :prison_law, :boolean
+      attribute :prior_authority_granted, :boolean
+      attribute :reason_why, :string
+      attribute :supporting_documents
+      attribute :next_hearing, :boolean
+      attribute :next_hearing_date, :date
+      attribute :client_detained, :boolean
+      attribute :subject_to_poca, :boolean
+      attribute :main_offence, :string
+      attribute :solicitor
+      attribute :plea, :string
+      attribute :court_type, :string
+      attribute :psychiatric_liaison, :boolean
+      attribute :psychiatric_liaison_reason_not, :string
+      attribute :youth_court, :boolean
 
-      def overview
-        rows(:laa_reference, :ufn, :prison_law_string)
+      def overview_card
+        OverviewCard.new(self)
       end
 
-      def prison_law_string
-        I18n.t("shared.#{prison_law}")
+      def primary_quote_card
+        PrimaryQuoteCard.new(self)
       end
 
-      def primary_quote_rows
-        []
+      def reason_why_card
+        ReasonWhyCard.new(self)
       end
 
-      private
+      def alternative_quotes
+        quotes.reject { _1['primary'] }.map { Quote.new(_1.merge(item_type:)) }
+      end
+
+      def alternative_quote_card(alternative_quote)
+        AlternativeQuoteCard.new(self, alternative_quote)
+      end
```

Comment:
as per comment in view - don;'t see the advantage of this being a separate function..

Reply:
Ah, see, what you miss is that the advantage is that this code didn't require me to be thinking clearly to _write_ it...

Yes, you're entirely right. Will fix.